### ISSUE TITLE
Need Clear Instructions for Enabling kubectl logs and kubectl exec

### ISSUE THREAD

**Issue by itselavia** [kubeedge/kubeedge#2740]
**What happened and what you expected to happen**:
I have a KubeEdge setup running and want to enable the logs and exec feature. But I am finding the [official docs](https://kubeedge.io/en/docs/advanced/debug/) confusing. I need help with figuring out which commands to run on which nodes.
So far I have the following nodes in my cluster:
1. Kubernetes Master Node
2. Kubernetes Worker Node
3. Cloudcore Node
4. Edgecore Node

The Step 1 and Step 2 in the [official docs](https://kubeedge.io/en/docs/advanced/debug/) mention finding the ca.crt and ca.key files, and setting the CLOUDCOREIPS variable. Are these two steps supposed to be performed on the Kubernetes master, cloud core, edge core, or all?

**How to reproduce it (as minimally and precisely as possible)**:
I have documented and automated the scripts to deploy KubeEdge on Google Cloud. The blog post is available here: https://itselavia.github.io/2021/01/02/deploy-kubeedge-gcp/

**Anything else we need to know?**:
I just need help figuring out where and how exactly to run the commands mentioned in Step 1, 2 and 3.
Basically, I would be helpful if the documentation mentioned at every step on which node(s) the corresponding commands have to be run.
I would love to submit a PR for the official docs once I finish this task. Thank you. 

**Environment**:
- Kubernetes version : 1.19.8
- KubeEdge version: v1.5.0

**Comment by daixiang0** [comment]
Sounds great, we need a deploy doc at Google Cloud.

It should be performed on cloudcore node, now cloudcore node is equal to k8s master node.

**Comment by ansjin** [comment]
@itselavia Did it work for you? For me when restarting the cloudcore after making the changes, I get the following error: 

`
I0608 01:32:11.342390  280680 server.go:73] Version: v1.7.0
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15df708]

goroutine 1 [running]:
github.com/kubeedge/kubeedge/cloud/cmd/cloudcore/app.NewCloudCoreCommand.func1(0xc00045bb80, 0x28be118, 0x0, 0x0)
        /root/codes/src/github.com/kubeedge/kubeedge/cloud/cmd/cloudcore/app/server.go:84 +0x2e8
github.com/spf13/cobra.(*Command).execute(0xc00045bb80, 0xc000118190, 0x0, 0x0, 0xc00045bb80, 0xc000118190)
        /root/codes/src/github.com/kubeedge/kubeedge/vendor/github.com/spf13/cobra/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc00045bb80, 0xc00010e058, 0x0, 0x0)
        /root/codes/src/github.com/kubeedge/kubeedge/vendor/github.com/spf13/cobra/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /root/codes/src/github.com/kubeedge/kubeedge/vendor/github.com/spf13/cobra/command.go:887
main.main()
        /root/codes/src/github.com/kubeedge/kubeedge/cloud/cmd/cloudcore/cloudcore.go:16 +0x65`


Environment:
Kubernetes version : 1.21.1
KubeEdge version: v1.7.0

**Comment by illyaSlobozhanin** [comment]
@itselavia I generated all steps on k8s master node and copy all certs to needed dirs on cloudcore-edgecore nodes.

**Comment by illyaSlobozhanin** [comment]
BTW. This is not working if cloudcore setuped via k8s deployment. (I think about k8s apiserver, routing and ssl
)
